README.md

```diff
@@ -23,15 +23,23 @@ another 2 weeks polishing for the first release. Since then development has pick
 down, with various different teams and maintainers. The current maintainer of the game and masterserver
 is "winrid" on GitHub.
 
-## Contributing and Reporting Bugs
-
-See [CONTRIBUTING.md](https://github.com/netpanzer/netpanzer/blob/master/CONTRIBUTING.md)
-
 ## To Play
 
 Go to the Releases tab in GitHub and download the version for your OS. The version in Linux repos as of 2024
 is very (8+yrs) old, so we suggest simply downloading the latest release directly.
 
+### Latest Download
+
+[You can download the latest version here.](https://github.com/netpanzer/netpanzer/releases/tag/0.9.0-RC-6)
```

Comment:
```suggestion
[You can download the latest version here.](https://github.com/netpanzer/netpanzer/releases/tag/0.9.0-RC-6). Alternatively, an  [Appimage](https://github.com/netpanzer/netpanzer/releases/tag/appimage-weekly) is available for Linux systems.
```

Reply:
Sure, let's add the app image to the readme once it works - it segfaults ATM right?

Reply:
It works fine if ran from where it's mounted, as mentioned at https://github.com/netpanzer/netpanzer/releases/tag/appimage-weekly

But if you wanna wait before it's added to the README, I'm fine with that